sdlib/d/gc/extent.d

```diff
@@ -516,6 +516,9 @@ void writePackedFreeSpace(ushort* ptr, ushort x) {
 	*ptr = value & ushort.max;
 }
 
+static assert(MaxSmallSize < 0x4000,
+              "Max small alloc size doesn't fit in 14 bits!");
+
```

Comment:
Move next to the explanations of the format, no? This is where we assert that the bullshit we describe int he comment acutally works :)